examples/src/main/java/io/grpc/examples/errorhandling/ErrorHandlingClient.java

```diff
@@ -136,6 +134,9 @@ public void onFailure(Throwable t) {
     if (!Uninterruptibles.awaitUninterruptibly(latch, 1, TimeUnit.SECONDS)) {
       throw new RuntimeException("timeout!");
     }
+    Verify.verify(status.getCode() == Status.Code.INTERNAL);
```

Comment:
We don't want to encourage this approach, since that requires users watching on the latch an having another thread sitting around. We shouldn't depend on the latch. We could log instead, or something like that.